src/Components/AddProduct/AddProduct.js

```diff
@@ -69,7 +70,12 @@ const AddProduct = (props) => {
 
         <Typography>Select Category</Typography>
         <br />
-        <Select options={options} onChange={(ev) => setCategory(ev.value)} />
+        <Select value={category} displayEmpty
+          inputProps={{ 'aria-label': 'Without label' }} onChange={(ev) => setCategory(ev.target.value)}>
```

Comment:
What is the use of this inputProp?
inputProps={{ 'aria-label': 'Without label' }}